cpu/native/include/async_read.h

```diff
@@ -29,7 +29,7 @@ extern "C" {
  * @brief   Maximum number of file descriptors
  */
 #ifndef ASYNC_READ_NUMOF
-#define ASYNC_READ_NUMOF 2
+#define ASYNC_READ_NUMOF 8
```

Comment:
Is this increase only necessary because 
```
CFLAGS += -DASYNC_READ_NUMOF=$(shell expr $(ZEP_DEVICES) + 1)
```
is removed from the makefile?

Reply:
Kinda, the default would now need to be +1
But then again why place such a low limit in `native` where a few bytes extra RAM use really don't matter.

Reply:
Yeah, on native it doesn't matter.